pennylane/labs/intermediate_reps/rowcol.py

```diff
@@ -0,0 +1,617 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""CNOT routing algorithm ROWCOL as described in https://arxiv.org/abs/1910.14478."""
+
+from typing import Iterable
+
+import galois
+import networkx as nx
+import numpy as np
+from networkx.algorithms.approximation import steiner_tree
+
+
+def postorder_traverse(tree: nx.Graph, source: int, source_parent: int = None):
```

Comment:
Are these public functions on porpouse?

Reply:
I think its actually good to have them public, because then the docs render and can be linked to to explain what postorder and preorder traversal are doing :)